src/components/etherscan-link/etherscan-link.tsx

```diff
@@ -48,13 +48,13 @@ export const EtherscanLink = ({
   copyToClipboard = CopyToClipboardType.NONE,
   ...props
 }: EtherscanLinkProps) => {
-  const { chainId } = useWeb3();
+  const { chainId } = useWeb3React();
   let hash: string;
   let href: string | null;
   const { t } = useTranslation();
   const { copy, copied } = useCopyToClipboard();
   const linkText = text ? text : t("View on Etherscan (opens in a new tab)");
-  const createLink = etherscanLinkCreator(chainId);
+  const createLink = etherscanLinkCreator(`0x${chainId}` as EthereumChainId);
```

Comment:
nitpick (non-blocking): If we are getting the chainId as a number should we change the type to be just a number?

Reply:
Lets do  this in a follow up pr